.github/workflows/auto-approve.yml

```diff
@@ -12,44 +12,11 @@ on:
 
 jobs:
   auto-approve:
-    if: contains(github.event.pull_request.labels.*.name, 'pr/auto-approve') || github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
-
+    if: contains(github.event.pull_request.labels.*.name, 'pr/auto-approve')
```

Comment:
I just reverted this workflow to its simple revision, before we even added special handling for dependabot